pretix_sepadebit/payment.py

```diff
@@ -198,10 +236,17 @@ def execute_payment(self, request: HttpRequest, payment: OrderPayment):
                 'iban': request.session['payment_sepa_iban'],
                 'bic': request.session['payment_sepa_bic'],
                 'reference': ref,
-                'date': due_date.strftime("%Y-%m-%d")
             }
+            reminder = SepaDueDate.REMINDER_OUTSTANDING if set_reminder else SepaDueDate.REMINDER_PROVIDED
+
+            due = SepaDueDate(date=due_date, reminder=reminder)
+            due.payment = payment
+            due.save()
+
             payment.confirm(mail_text=self.order_pending_mail_render(payment.order))
         except Quota.QuotaExceededException as e:
+            if due is not None:
```

Comment:
In What situation can ``due`` be ``None``?

Reply:
Thinking about it, never.